rclpy/rclpy/qos.py

```diff
@@ -20,8 +20,43 @@
 from rclpy.impl.implementation_singleton import rclpy_implementation as _rclpy
 
 
+class QoSPolicyKind(IntEnum):
+    """
+    Enum for types of QoS policies that a Publisher or Subscription can set.
+
+    This enum matches the one defined in rmw/incompatible_qos_events_statuses.h
+    """
+
+    RMW_QOS_POLICY_INVALID = 1 << 0
```

Comment:
It would also be good to use the values defined in `rmw`, instead of manually write them here again.

I don't mind strongly, you can left a TODO.

Reply:
I am not sure how to do that. Do you have an example?

`QoSPublisherEventType` and `QoSSubscriptionEventType` were also done this way.

Reply:
Not really nice, but here an example: https://github.com/ros2/rclpy/blob/3d1698a99cbde82322c82fde428c76dd2242e855/rclpy/src/rclpy/_rclpy_logging.c#L244

You could also create the constants in the [module init function](https://github.com/ros2/rclpy/blob/3d1698a99cbde82322c82fde428c76dd2242e855/rclpy/src/rclpy/_rclpy.c#L5516), but again, not super nice.

Reply:
I see. I'm not sure if this is worth the effort, because it won't automatically synchronize when new enums are added to `rmw_qos_policy_kind_t`.

I'll leave a `TODO` here.

Reply:
> I see. I'm not sure if this is worth the effort, because it won't automatically synchronize when new enums are added to rmw_qos_policy_kind_t.

Sounds good!